src/Error/Error.php

```diff
@@ -330,6 +330,8 @@ public function getExtensions()
      * @deprecated Use FormattedError::createFromException() instead
      *
      * @return mixed[]
+     *
+     * @codeCoverageIgnore
```

Comment:
what is the reason for these?

Reply:
Well, they're deprecated and unused, so they are falsely impacting the code coverage score. This was just a convenient way for me to increase the overall average coverage score so that Scrutinizer would pass.

Reply:
Oh I didn't see they're deprecated. In that case I think it's ok. 

But generally Travis should always pass, scrutinizer is nice to have. If coverage decreases it doesn't necessarily be bad/issue. It's just a simple metric anyways.

Reply:
And if we're not all that interested in scrutinizer's static checking, I would highly recommend replacing it with PHPStan. Trying to satisfy a checker that is black box, not open source, not available to run locally, and full of bugs is a nightmare.